core/src/autogluon/core/models/abstract/abstract_model.py

```diff
@@ -932,6 +931,31 @@ def _get_default_stopping_metric(self):
         stopping_metric = metrics.get_metric(stopping_metric, self.problem_type, 'stopping_metric')
         return stopping_metric
 
+    def _get_params(self) -> dict:
+        """Gets all params."""
+        return self.params.copy()
+
+    def _get_ag_params(self) -> dict:
+        """Gets params that are not passed to the inner model, but are used by the wrapper."""
```

Comment:
'inner model' may not be clear language for contributors

Reply:
any recommendation for improved wording?

Reply:
maybe just: inner model (`self.model`)?

Reply:
added